src/app/views/dashboard/stickers/stickers.ts

```diff
@@ -0,0 +1,48 @@
+import Component from 'vue-class-component';
+import { MediaItem } from '../../../../_common/media-item/media-item-model';
+import { BaseRouteComponent } from '../../../../_common/route/route-component';
+import { Sticker } from '../../../../_common/sticker/sticker.model';
+import AppPageHeader from '../../../components/page-header/page-header.vue';
+
+export type InitPayload = {
+	value: number;
+	stickerCounts: StickerCountPayload[];
+	stickers: any[];
+	stickerCost: number;
+};
+
+type StickerCountPayload = {
+	sticker_id: string;
+	count: string;
+};
+
+export type StickerCount = {
+	sticker_id: number;
+	count: number;
+	sticker: Sticker;
+};
+
+@Component({
+	name: 'RouteDashStickers',
+	components: {
+		AppPageHeader,
+	},
+})
+export default class RouteDashStickers extends BaseRouteComponent {
+	_coverMediaItem?: MediaItem;
+
+	get coverMediaItem() {
```

Comment:
No need for the _coverMediaItem. Vue will cache this since it's a component getter and will never call it again.